sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -344,6 +344,12 @@ export interface CreateKeyOptions extends coreHttp.OperationOptions {
    * Size of the key
    */
   keySize?: number;
+  /**
+   * The retention dates of the softDelete data.
+   * The value should be >=7 and <=90 when softDelete enabled.
```

Comment:
Is this generated? If not, should probably be "soft delete" - though, be consistent throughout (i.e. don't change this if you have "softDelete" everywhere else).

Reply:
We have it in several places, but I can change them: https://github.com/Azure/azure-sdk-for-js/search?q=softdelete&unscoped_q=softdelete

Reply:
If it's consistently everywhere I wouldn't.